02/Form.js

```diff
@@ -1,11 +1,25 @@
 import React from 'react';
 
 class Form extends React.Component {
+    handleSubmit = (event) => {
+        event.preventDefault();
+        const name = this.nameInput.value.trim();
+        const surname = this.surnameInput.value.trim();
+
+        if (name && surname) {
+            this.props.onAddUser({ name, surname });
```

Comment:
Można też wykorzystać destrukturyzację dla prosp tj. `const {onAddUser} = this.props` na początku metody :)